acceptance/suites/pre_suite/puppet4_compat/80_configure_agents.rb

```diff
@@ -0,0 +1,13 @@
+require 'puppetserver/acceptance/compat_utils'
```

Comment:
Do we need to have any of these steps be run for a Puppet 4 agent?  I was thinking these were only needed for a Puppet 3 agent.